pandas/tests/resample/test_resample_api.py

```diff
@@ -405,7 +405,7 @@ def test_agg_misc():
     ]
 
     # passed lambda
-    for t in cases:
+    for i, t in enumerate(cases):
```

Comment:
Why? Where is this `i` used?

Reply:
this was in debugging, thought i had reverted it, thanks for pointing it out